app/models/post.rb

```diff
@@ -5,6 +5,11 @@ class Post < ApplicationRecord
   after_save :update_post
   after_destroy :decrement_likes
 
+  validates :title, presence: true
+  validates :title, length: { maximum: 250 }
+  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
```

Comment:
- [ ] I highlighted this because I couldn't find `post controller` but consider working on flash messages for a successful and not successful path `posts#create`

Reply:
- [ ] Check your flash messages in the post controller are not working when I create a new post I am supposed to get a message
![flash](https://user-images.githubusercontent.com/44745855/151658803-d1674621-737b-478b-84ff-068bef0312f7.png)

Reply:
- [ ] Consider working on your N+1 problem when fetching all posts and their comments for a user as shown instead of doing this
![n+1](https://user-images.githubusercontent.com/44745855/151658956-6a0d5a22-d7d0-490d-819b-d9936977fac1.png)

you can do `@posts = @user.posts.includes(:comments)`